src/Reflex/Class.hs

```diff
@@ -422,6 +420,8 @@ class MonadSample t m => MonadHold t m where
   -- | Create a new 'Event' that only occurs only once, on the first occurrence of
   -- the supplied 'Event'.
   headE :: Event t a -> m (Event t a)
+  -- | An event which occurs at the current moment.
+  now :: m (Event t ())
```

Comment:
Perhaps it would make sense to add a default signature based on MonadTrans so that all these new methods can be added automatically.

Reply:
Great idea, I've added it.